code/modules/events/headcrabs.dm

```diff
@@ -35,7 +34,7 @@
 
 /datum/event/headcrabs/start()
 
-	location = rand(0,19)
+	location = rand(0,17)
```

Comment:
I'm really not sure how I feel about hardcoding a long list of pre-defined /areas on this event.
What if we use an alternate map which doesn't have one of these areas, and that area is rolled? What happens?